pkg/cable/libreswan/libreswan.go

```diff
@@ -150,15 +150,9 @@ func (i *libreswan) refreshConnectionStatus() error {
 
 // GetActiveConnections returns an array of all the active connections for the given cluster.
 func (i *libreswan) GetActiveConnections(clusterID string) ([]string, error) {
-	if err := i.refreshConnectionStatus(); err != nil {
-		return []string{}, err
-	}
-
```

Comment:
@mangelajo, just for info - Is there an issue if we call refreshConnectionStatus?